src/Files.App/Views/Shells/BaseShellPage.cs

```diff
@@ -142,6 +142,18 @@ public bool IsCurrentInstance
 						_IsCurrentInstanceTCS = new();
 
 					NotifyPropertyChanged(nameof(IsCurrentInstance));
+
+					if (!IsColumnView)
```

Comment:
What about Column View?

Reply:
Avoided to getting notified from column focus change.

I strongly suppose this shell concept is instantaniously away from the earth, the codebase makes us to implement extensibly. I'll keep stick around this.

Reply:
What's the plan to change the background color in that case?

Reply:
Wdym?

Reply:
The active pane background color doesn't update if one of the panes is Columns View.

Reply:
Never mind